conversions/decimal_to_binary_recursion.c

```diff
@@ -0,0 +1,23 @@
+#include <assert.h>
```

Comment:
please update the beginning of the file with a structure as mentioned in the [guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#typical-structure-of-a-program)